rules/docker-eventd.mk

```diff
@@ -21,9 +21,6 @@ $(DOCKER_EVENTD)_INSTALL_DEBS = $(PYTHON3_SWSSCOMMON)
 $(DOCKER_EVENTD)_VERSION = 1.0.0
 $(DOCKER_EVENTD)_PACKAGE_NAME = eventd
 
-$(DOCKER_DHCP)_SERVICE_REQUIRES = updategraph
```

Comment:
>updategraph [](http://example.com/codeflow?start=34&length=11)

require config-setup?

Reply:
This makefile is wrong, it should be DOCKER_EVENTD, $(DOCKER_DHCP)_SERVICE_REQUIRES does not make any sense.

Reply:
@zbud-msft Could you review here?

Reply:
Agreed this should not be in docker-eventd.mk, thanks for fix